docs/tracing/getting-started.md

```diff
@@ -0,0 +1,157 @@
+title: Getting Started with Sematext Tracing
+description: Learn how to set up distributed tracing with OpenTelemetry and Sematext
+
+## Getting Started with Sematext Tracing
+
+Sematext Tracing helps you monitor and troubleshoot distributed applications by tracking requests as they flow through your services. Built on OpenTelemetry, it provides visibility into performance bottlenecks, error propagation, and service dependencies.
+
+## Prerequisites
+
+Before you begin, ensure you have:
+
+- A Sematext Cloud account ([sign up free](https://apps.sematext.com/ui/registration))
+- Access to your application's source code or deployment configuration
```

Comment:
I think the emphasis here is on OR, right? As a dev(ops) or SRE I would much rather have to change only the deployment config than have to touch the source code, which implies having to run new builds, ensuring I didn't break anything, more testing in non-prod, etc. And, as a matter of fact, if you choose to auto-instrument your applications, then you absolutely don't have to touch the applications (source code) themselves. So I would really emphasize this sort of stuff here to make that clear.